Sources/TuistSigning/SigningInstaller.swift

```diff
@@ -46,16 +38,25 @@ final class SigningInstaller: SigningInstalling {
         self.securityController = securityController
     }
 
-    func installProvisioningProfile(_ provisioningProfile: ProvisioningProfile) throws {
-        guard provisioningProfile.expirationDate > Date() else { throw SigningInstallerError.expiredProvisioningProfile(provisioningProfile) }
+    func installProvisioningProfile(_ provisioningProfile: ProvisioningProfile) throws -> [LintingIssue] {
+        var issues = [LintingIssue]()
+
+        if provisioningProfile.expirationDate < Date() {
+            issues.append(.expiredProvisioningProfile(provisioningProfile))
+            return issues
```

Comment:
need to return here or otherwise test would fail on Line 51: https://github.com/tuist/tuist/pull/2919/files#diff-bce310dbe3560d414241ef1ddfd527bb1981be570f3b9ccaba68ae15e9dfda2dR51

Reply:
I'd add a comment here then explaining that we don't want to create a directory for a provisioning profile that has expired, so that's why we return early.

Reply:
I actually just returned because of failing unit tests.

I decided it's better to install the provisioning profile in any case, so Xcode will also emit a warning. Had to adapt the unit tests a bit.